Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -699,6 +704,48 @@ static void replaceContentsInRange(WebCore::Document& document, const WebCore::S
     m_webPage->textReplacementSessionShowInformationForReplacementWithUUIDRelativeToRect(data.sessionUUID, data.uuid, rect);
 }
 
+
+void UnifiedTextReplacementController::getTextIndicatorForID(const WTF::UUID& uuid, CompletionHandler<void(std::optional<WebCore::TextIndicatorData>&&)>&& completionHandler)
```

Comment:
Is there a reason the completion handler takes an optional? Because of your early return on line 741, the optional will never be null.

Reply:
Never mind 😅